front-end/src/components/Tables/SubmitTable.js

```diff
@@ -1,11 +1,218 @@
-import React from "react";
+import TextField from '@mui/material/TextField';
+import Container from '@mui/material/Container';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import { Dialog, DialogTitle, DialogContent, DialogContentText, Link } from '@mui/material';
+import React, { useState } from 'react';
+import axios from "axios";
 
 const SubmitTable = () => {
+  const [locationName, setLocationName] = useState("")
```

Comment:
I believe we can refactor this in the future so that it uses one `formData` state object that keeps track of all the form inputs. I think for now this is fine though.